Source/towners.cpp

```diff
@@ -1157,17 +1141,18 @@ void __fastcall TalkToTowner(int p, int t)
 void __fastcall CowSFX(int pnum)
 {
 	if (CowPlaying == -1 || !effect_is_playing(CowPlaying)) {
-		if (sgdwCowClicks++ < 8) {
-			CowPlaying = (sgdwCowClicks == 4) + TSFX_COW1;
-		} else {
+		sgdwCowClicks++;
+		if (sgdwCowClicks >= 8) {
 			PlaySfxLoc(TSFX_COW1, plr[pnum].WorldX, plr[pnum].WorldY + 5);
 			sgdwCowClicks = 4;
 			CowPlaying = snSFX[sgnCowMsg][plr[pnum]._pClass]; /* snSFX is local */
-			if (sgnCowMsg++ >= 2)
+			sgnCowMsg++;
+			if (sgnCowMsg >= 3)

```

Comment:
How does pre- and post-increment work? I would have guessed that the post-increment would be equal to `>= 2` and not `>= 3`, so my understanding of this most likely needs refreshment.

Reply:
The 2 that was there was actually wrong, would have had to be changed to 3 even if the statement had stayed on one line

Reply:
Perfect, then I understand. Thanks!